ci/scripts/run_tests.py

```
@@ -217,11 +238,11 @@ def main(junit_xml: str | None,
     with ProcessPoolExecutor(max_workers=jobs) as executor:
         ex = executor
 
-        def shutdown_executor(signum, frame):
+        def shutdown_executor(signum, frame, wait: bool = False):
```

Ruff (0.15.2)

[warning] Unused function argument: `signum`

(ARG001)